controllers/depresolver/depresolver_config.go

```diff
@@ -257,13 +343,13 @@ func parseLogOutputFormat(value string) LogFormat {
 func (c *Config) GetExternalClusterNSNames() (m map[string]string) {
 	m = make(map[string]string, len(c.ExtClustersGeoTags))
 	for _, tag := range c.ExtClustersGeoTags {
-		m[tag] = getNsName(tag, c.DNSZone, c.EdgeDNSZone, c.EdgeDNSServer)
+		m[tag] = getNsName(tag, c.DNSZone, c.EdgeDNSZone, c.EdgeDNSServers[0].Host)
 	}
 	return
 }
```

Comment:
EdgeDNS server has no influence on NS name (see [this](https://github.com/k8gb-io/k8gb/blob/master/controllers/depresolver/depresolver_config.go#L295-L297)), maybe it would be enough to add validation that would look for EDGE_DNS_SERVERS can be only 127.0.0.1 or localhost, or server list (except 127.0.0.1, localhost)

Reply:
It would simply pick the first one and never fallback

Reply:
edgedns server has a direct influence on NS name resolution. Sticking with only one defeats the idea of multiple edgeDNS Servers to rely on unless I miss something

Reply:
@jkremser actually i recalled the `getNSName` logic, it's fine to evaluate the first one for local scenario. Not sure how it is obvious for external guys though

Reply:
I've added the check that would fail if localhost or 127.0.0.1 appears on some other position in the list + test called `TestResolveConfigWithMultipleEdgeDnsServersLocalhostNotFirst`

Reply:
right, the above example will pass the validation (as I've done it) and I am not sure if it should or should not be supported as a valid use-case. Either way I am not sure how the `getNSName` should behave in this case. Perhaps instead of passing the `c.EdgeDNSServers[0].Host` to that method, we can pass a boolean called `isLocal` or something similar with the intention of not doing the string concat logic in that method, but returning localhost instead... well and that boolean can be calculated as true iff the list contains only 1 element and the `element.Host == localhost / 127.0.0.1`

wdyt? that way we will still have the fallback mechanism for other methods (Dig, Exchange)